dm/checker/checker.go

```diff
@@ -217,7 +218,13 @@ func (c *Checker) Init(ctx context.Context) (err error) {
 				if err != nil {
 					return err
 				}
-				c.checkList = append(c.checkList, checker.NewSourceDumpPrivilegeChecker(instance.sourceDB.DB, instance.sourceDBinfo, checkTables, exportCfg.Consistency))
+				c.checkList = append(c.checkList, checker.NewSourceDumpPrivilegeChecker(
+					instance.sourceDB.DB,
+					instance.sourceDBinfo,
+					checkTables,
+					exportCfg.Consistency,
+					false,
```

Comment:
```suggestion
				c.dumpWholeInstance,
```
I don't understand why it is not `c.dumpWholeInstance `

Reply:
I should add an unit test at this level...

Reply:
https://github.com/pingcap/tiflow/pull/7331/commits/f7a9ceda8304b9d7e4465b1b73e85b17624c80a2